pkgs/shells/powershell/getHashes.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p bash wget coreutils gnutar nix
+version=$1
+
+if [[ -z "$1" ]]
+then
+    echo "Pass the version to get hashes for as an argument"
+    exit 1
+fi
+
+allOutput=""
+
+for plat in osx linux
+do
+    for arch in x64 arm64
+    do
+        dlDest=$(mktemp)
+        exDest=$(mktemp -d)
+
+        URL="https://github.com/PowerShell/PowerShell/releases/download/v$version/powershell-$version-$plat-$arch.tar.gz"
+        wget $URL -O $dlDest >&2
+
+        tar -xzf $dlDest -C $exDest >&2
+
+        hash=$(nix hash path $exDest)
+
+        output=""
+        output+=$'\n'"$plat $arch"
+        output+=$'\n'"$hash"
+        output+=$'\n\n\n'
+        echo "$output" >&2
```

Comment:
Wouldn't that duplicate all output?

Reply:
I removed it

Reply:
I originally had the idea to have it after each download, but wget is too verbose, so it got lost, so I added it at the end and moved the intermittent output to stderr. But whatever